core/tests/puppeteer-acceptance-tests/specs/logged-out-user/visit-classroom-index-page.spec.ts

```diff
@@ -44,8 +61,47 @@ describe('Logged-out User', function () {
     DEFAULT_SPEC_TIMEOUT_MSECS
   );
 
-  // TODO (#20610): Add test for one and more than one classroom.
-  // Once issue with relase coordinator user is fixed.
+  it(
+    'should be redirected to a classroom page if we have 1 public classroom.',
+    async function () {
+      await curriculumAdmin.navigateToTopicAndSkillsDashboardPage();
+      await curriculumAdmin.createTopic('Test Topic 1', 'test-topic-one');
+      await curriculumAdmin.createNewClassroom('Math', 'math');
+      await curriculumAdmin.updateClassroom(
+        'Math',
+        'Teaser Text',
+        'Course Details',
+        'Topic List Intro'
+      );
+      await curriculumAdmin.addTopicToClassroom('Math', 'Test Topic 1');
+      await curriculumAdmin.publishClassroom('Math');
+
+      await loggedOutUser.navigateToClassroomsPage();
+      await loggedOutUser.expectToBeOnClassroomPage('Math');
```

Comment:
We have a function expectToBeOnPag(), please use that.

Reply:
Yes makes sense but we need to make sure that the current classroom page is opened and content is visible for a public classroom.